build.gradle

```diff
@@ -74,7 +74,6 @@ subprojects {
         skipExistingHeaders true
         strictCheck true
         include "**/*.java"
-        excludes(["**/gen-java/**"])
```

Comment:
It seems that it is supported https://github.com/hierynomus/license-gradle-plugin, however it does not work here. I have disabled license check for the whole jaeger-thrift module